src/AnnotatedCommand.php

```diff
@@ -376,11 +389,23 @@ protected function initialize(InputInterface $input, OutputInterface $output)
         $this->commandProcessor()->initializeHook($input, $this->getNames(), $this->annotationData);
     }
 
+    protected function executeCallsCommands() {
+        if ($this->getCalls()) {
+            foreach ($this->getCalls as $command_name) {
+                // @todo validate that $command_name exists in this application.
```

Comment:
This is the interesting part. The $application field of Symfony Console's command class is private. 😢 

We could override setApplication to cache our own copy of it. Not super great.